tasks/scrapeData/cleanLocations.js

```diff
@@ -0,0 +1,39 @@
+const scraperVars = ['type', 'timeseries', 'headless', 'ssl', 'priority', 'aggregate'];
+
+/*
+  Remove "private" object properties
+*/
+const removePrivate = location => {
```

Comment:
Same as in the next file.

Also may as well do `if empty or underscore` rather than write the `delete` twice

Reply:
Good call, though refactoring I ended up with separate conditionals, they should be combined to one expression.